src/helpers/Query.ts

```
@@ -183,6 +184,110 @@
+
+			if (Array.isArray(request['relations'])) {
+				relationsArray = request['relations']
+			}
+			// if it's a string, convert it to an array
+			else if (typeof request['relations'] === 'string') {
+				relationsArray = request['relations'].split(',')
+			}
+
+			// convert relations to DataSourceSchemaRelation[]
+
+			for(const relation of relationsArray) {
```

ESLint

[error] Insert `·`

(prettier/prettier)